authors/apps/articles/migrations/0002_auto_20190415_1518.py

```diff
@@ -1,5 +1,6 @@
-# Generated by Django 2.2 on 2019-04-11 13:57
+# Generated by Django 2.2 on 2019-04-15 15:18
```

Comment:
@dannylwe do not merge migrations

Reply:
@ja-odur hi, it looks like I will have to manually copy over the migrations into empty files. I tried and not figured out another efficient way to do so.